Abstracted contact info

TiCampbell opened the pull request:
Please review and merge #331 **first**.

There was need to change the contact number on the site, which led to abstracting contact information into `app.constants`, as well as using the `TWILIO_NUMBER` secret as the user-facing phone number (to simplify by only have one contact number throughout the app). Also included are instruction to set the Twilio number to forward to a physical number.

penoptech commented on README.md:
```diff
@@ -462,6 +468,61 @@ Connect to the database to configure and complete the following steps:
+
+1. From the initial screen once logged in to Twilio, select 'Explore Products +'.
+
+1. Under the Developer Tools section, find 'Functions and Assets'. Select the pushpin icon to pin in to the Twilio sidebar navigation.
```
Not that this is anything of substance... The configuration steps for call-forwarding all use '1.'

penoptech reviewed (commented):
This is awesome! Now we don't have to worry about updating the contact email and phone number in a bunch of different places throughout the application. Great work!

Just a couple of questions:
- This is just purely curiosity on my part. Why have the twillio number in `settings.py`, but the email in `constants.py`?
- Did you consider passing the context variables for `contact_number` and contact_email` through the `global_template_variables` function in `context_processors.py`? That way they're available in all of the templates and there wouldn't be a need for explicitly return them through the view functions.

TiCampbell commented:
@penoptech Great points! 

- I didn't really give it any thought - the Twilio number already existed in `.env` so I left it alone, and the email was a new constant so I added it to `constants`. They should definitely be in the same place, which is likely the future universal settings table (although the Twilio number is so intertwined with the code functionality I worry about it being changed in the table and just crashing everything). For now, to keep them out of source control, I moved the contact email to `.env` along with the Twilio number.
- I totally forgot about `context_processors`! That's a much better place; they're moved over now.

Please take a look at the changes and let me know. Thanks!

penoptech reviewed (approved):
@TiCampbell, looks great!
